coding-concepts.js

```diff
@@ -53,8 +52,8 @@ var myCodingSkills = {
 }
 // console.log(myCodingSkills.languages[0])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "JavaScript"
+// b) Verify and explain: Because it is an array that holds collection of data (JavaScript object) I am able to invoke the object independent of the order that it is and it will bring me back the value that I asked. 
```

Comment:
The array is a value of the object with the key language.  In this case, the object is assigned to the variable myCodingSkills,  where you can access the data using dot notation and accessing a specific index inside the array.